src/test/enginesynctest.cpp

```diff
@@ -1340,9 +1340,9 @@ TEST_F(EngineSyncTest, SyncPhaseToPlayingNonSyncDeck) {
     ControlObject::getControl(ConfigKey(m_sGroup1, "play"))->set(1.0);
     ProcessBuffer();
 
-    EXPECT_FLOAT_EQ(140.0,
+    EXPECT_FLOAT_EQ(100.0,
```

Comment:
What this is testing is that if there is a playing deck with master sync enabled, we should choose that as the sync target.  That seems like a different change from what the bug was saying.  If a deck is playing with sync, it should definitely be the thing that other decks sync to.  So this test change seems wrong to me